README.md

```diff
@@ -14,6 +14,36 @@ For long-term goals and direction, please see [`VISION.md`](https://github.com/p
 * Swift 4.0+
 * iOS 9+
 
+## Getting Started Guide
+
+Please see our [Getting Started](https://github.com/plangrid/ReactiveLists/blob/master/Guides/Getting%20Started.md) guide to learn how to use ReactiveLists
+
+
+#### `SectionModel`
```

Comment:
We probably don't want to dupe this stuff. I think we should keep the link above and have that be the single source of truth.

Reply:
(and remove all this)

Reply:
oops yeah I thought I got rid of that from `README`.  Will drop it